glide-core/src/client/value_conversion.rs

```diff
@@ -640,6 +687,256 @@ mod tests {
         assert!(converted_4.is_err());
     }
 
+    #[test]
+    fn convert_xread() {
```

Comment:
please add a test that map with none key_type and value_type doesn't change the inner values